src/components/Layout/ContainerWithHeader.js

```diff
@@ -94,9 +95,16 @@ class ContainerWithHeader extends React.Component<Props> {
     StatusBar.setBarStyle('dark-content');
   };
 
-  renderContent = (isShortScreenWithFooter) => {
+  renderContent = (isShortScreenWithFooter, shouldRenderChildrenInScrollView) => {
     const { children, keyboardAvoidFooter } = this.props;
-    if (!isShortScreenWithFooter) return children;
+    if (!isShortScreenWithFooter && !shouldRenderChildrenInScrollView) return children;
+    if (!isShortScreenWithFooter && shouldRenderChildrenInScrollView) {
+      return (
+        <ScrollView style={{ flex: 1 }}>
+          {children}
+        </ScrollView>
+      );
+    }
```

Comment:
What do you think about having the first condition of both `if`s into its own `if`? I seems like it's claner to me, but not sure.

Something like:
```js
    if (!isShortScreenWithFooter) {
      if (!shouldRenderChildrenInScrollView) {
        return children;
      }

      return (
        <ScrollView style={{ flex: 1 }}>
          {children}
        </ScrollView>
      );
    }
```

Reply:
Sure :) Will adjust